x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.tsx

```diff
@@ -45,49 +57,107 @@ export const isCommonError = (error: unknown): error is CommonError => {
   return true;
 };
 
-const packageNotInstalledRenderer = (cisIntegrationLink?: string) => (
-  <FullSizeCenteredPage>
-    <NoDataPage
-      data-test-subj={PACKAGE_NOT_INSTALLED_TEST_SUBJECT}
-      css={css`
-        max-width: 950px;
-      `}
-      pageTitle={i18n.translate('xpack.csp.cloudPosturePage.packageNotInstalled.pageTitle', {
-        defaultMessage: 'Install Integration to get started',
-      })}
-      solution={i18n.translate('xpack.csp.cloudPosturePage.packageNotInstalled.solutionNameLabel', {
-        defaultMessage: 'Cloud Security Posture',
-      })}
-      docsLink={'https://ela.st/kspm'}
-      logo={'logoSecurity'}
-      actions={{
-        elasticAgent: {
-          href: cisIntegrationLink,
-          isDisabled: !cisIntegrationLink,
-          title: i18n.translate('xpack.csp.cloudPosturePage.packageNotInstalled.buttonLabel', {
-            defaultMessage: 'Add a KSPM integration',
-          }),
-          description: (
+export interface CspNoDataPageProps {
+  pageTitle: NoDataPageProps['pageTitle'];
+  docsLink: NoDataPageProps['docsLink'];
+  actionHref: NoDataPageProps['actions']['elasticAgent']['href'];
+  actionTitle: NoDataPageProps['actions']['elasticAgent']['title'];
+  actionDescription: NoDataPageProps['actions']['elasticAgent']['description'];
+  testId: string;
+}
+
+export const CspNoDataPage = ({
+  pageTitle,
+  docsLink,
+  actionHref,
+  actionTitle,
+  actionDescription,
+  testId,
+}: CspNoDataPageProps) => (
+  <NoDataPage
+    data-test-subj={testId}
+    css={css`
+      max-width: 950px;
+    `}
+    pageTitle={pageTitle}
+    solution={i18n.translate('xpack.csp.cloudPosturePage.packageNotInstalled.solutionNameLabel', {
+      defaultMessage: 'Cloud Security Posture',
+    })}
+    docsLink={docsLink}
+    logo="logoSecurity"
+    actions={{
+      elasticAgent: {
+        href: actionHref,
+        isDisabled: !actionHref,
+        title: actionTitle,
+        description: actionDescription,
+      },
+    }}
+  />
+);
+
+const packageNotInstalledRenderer = (
+  kspmIntegrationLink?: string,
+  cspmIntegrationLink?: string
+) => {
```

Comment:
consider passing an object instead. once we add `vuln management`, it'd be nicer than passing a third argument